fftw3.yml

```diff
@@ -0,0 +1,15 @@
+name: fftw3
```

Comment:
fftw3f is in shared modules, and it build with the SIMD for the supported platforms. (they are opt-in not opt-out)

Reply:
Thanks, you are right! Here is it: https://github.com/flathub/shared-modules/blob/master/linux-audio/fftw3f.json

Reply:
Although it is a little outdated.

Reply:
merged. thanks. I was meaning to do it but got sidetracked.

Reply:
Now using FFTW3 from shared modules.